ls1intum/Artemis
# Bugfix/submission overview reconnect results

### Checklist

### Motivation and Context


The submission overview for a student does not show the correct result for failed builds, because the result is not connected to the submission on this page. Because the submission holds the `buildFailed` property, the result only displayed a `No tests found` status.

### Description

I reconnected the result to the submission in the service API call. This way, the submission is always available when displaying the result.

### Steps for Testing


1. Log in to Artemis
2. Navigate to the scores of a programming exercise
3. Click on the name of a student with failed builds
4. On the next submissions overview, the failed results should get displayed correctly

E.g. https://artemistest.ase.in.tum.de/#/participation/163480/submissions
